src/tasks/trigger/AutoInteractionTask.py

```
@@ -25,19 +25,14 @@ def run(self):
         now = self.next_frame()
         if self.config.get('自动跳过剧情', True):
             if self.find_one(fL.skip_dialog_esc, horizontal_variance=0.05, frame=now):
-                self.press_esc()
-                time.sleep(0.1)
+                self.log_info("检测到可跳过对话，发送 ESC")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)